Packs/ThousandEyes/Integrations/ThousandEyes/ThousandEyes.py

```diff
@@ -0,0 +1,758 @@
+
+    for event in events_to_create:
+
+        incident_created_time = datetime.utcnow().strftime(THOUSAND_EYES_DATE_TIME_FORMAT + "Z")
+        incident = {
+            "name": f"ThousandEyes Incident - {event.get('ruleName')} - {event.get('dateStart')}",
+            "occured": incident_created_time,
+            "rawJSON": json.dumps(event)
+        }
+
+        incidents_to_create.append(incident)
+
+    # Update last run and remove the last 'Z' from datetime string
+    last_fetch = incident_created_time[:-1]
+
+    next_run = {"start_time": last_fetch}
+    return next_run, incidents_to_create
+
+
+def get_alerts_command(
+    client: Client,
+    aid: Optional[int] = None,
+    from_date: Optional[str] = None,
+    to_date: Optional[str] = None,
+    fetch_for_incident: bool = False
+):
+    """
+        Fetches live alerts
+
+    Args:
+        client (Client Object): Client object with request.
+        aid (int) (Optional): Fetch Active Alerts for a specific AID.
+        from_date (str) (Optional): Explicit start date to fetch Alerts from.
+        to_date (str) (Optional): Explicit end date to fetch Alerts to.
+        fetch_for_incident (Boolean) (Default - False): Boolean flag to specify if the alerts are fetched for incidents or not
+
+    Returns:
+        Results of CommandResults type or List of raw_response (if called from fetch_incidents_command() function)
+
+    """
+
+    endpoint = '/alerts.json'
+    method = 'GET'
+    raw_response_list: List[Dict] = []
+    entry_context: List[Dict] = []
+    human_readable_data: List[Dict] = []
+
+    params: Dict = {}
+    if aid and not from_date and not to_date:
+        params = {
+            "aid": aid
+        }
+
+    elif aid and from_date and not to_date:
+        params = {
+            "aid": aid,
+            "from": from_date
+        }
+
+    elif aid and from_date and to_date:
+        params = {
+            "aid": aid,
+            "from": from_date,
+            "to": to_date
+        }
+
+    if fetch_for_incident:
+        params = {
+            "from": from_date
+        }
+        group_aid_list = fetch_all_aids(client)
+
+        if isinstance(group_aid_list, dict):
+            err_msg = group_aid_list
+            raise Exception(err_msg)
+
+        for group_aid in group_aid_list:
+            response = client.handle_rate_lmiit_and_make_request(
+                method,
+                endpoint,
+                params=params
+            )
+
+            alerts_found, raw_response_temp_list, entry_context_temp, human_readable_data_temp = parse_alerts_response(
+                client=client,
+                method=method,
+                response=response,
+                aid=group_aid,
+                human_readable=False
+            )
+
+            for raw_response_temp in raw_response_temp_list:
+                raw_response_list.append(raw_response_temp)
```

Comment:
```suggestion
            raw_response_list = raw_response_temp_list
```

Reply:
The for loop is required in this case, as the Raw API Response is to be fetched for all the different AID and appended as a separate object in the main "raw_response_list". So as to facilitate the parsing process.